modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Installing Istio
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+
+=== Install Istio 1.6.x
+
+Anypoint Service Mesh supports Istio versions `1.6.0` through `1.6.8`. To install and configure Istio 1.6.x for Anypoint Service Mesh:
```

Comment:
```suggestion
Anypoint Service Mesh supports Istio 1.6.x versions 1.6.0 through 1.6.8. To install and configure Istio 1.6.x for Anypoint Service Mesh:
```

Reply:
Consider whether there are prereqs for installing 1.6.x. 
If so, document them following the first sentence above (==== Prerequisites) and then use the subtitle ==== Steps to transition from prereqs to steps.

Reply:
Done! Thanks!